frontend/MainCode.js

```diff
@@ -25,20 +27,29 @@ export default function MainCode() {
   const context = useContext(MainContext);
 
   const [loaded] = useFonts({
-    'Inter-Regular': require('./assets/fonts/Inter-Regular.ttf'),
-    'Oxygen-Regular': require('./assets/fonts/Oxygen-Regular.ttf'),
-    'Inter-Bold': require('./assets/fonts/Inter-Bold.ttf'),
-    'Oxygen-Bold': require('./assets/fonts/Oxygen-Bold.ttf'),
-    'Inter-Light': require('./assets/fonts/Inter-Light.ttf'),
-    'Oxygen-Light': require('./assets/fonts/Oxygen-Light.ttf'),
+    "Inter-Regular": require("./assets/fonts/Inter-Regular.ttf"),
+    "Oxygen-Regular": require("./assets/fonts/Oxygen-Regular.ttf"),
+    "Inter-Bold": require("./assets/fonts/Inter-Bold.ttf"),
+    "Oxygen-Bold": require("./assets/fonts/Oxygen-Bold.ttf"),
+    "Inter-Light": require("./assets/fonts/Inter-Light.ttf"),
+    "Oxygen-Light": require("./assets/fonts/Oxygen-Light.ttf"),
   });
   const [popupAlert, setPopupAlert] = useState(false);
 
   return (
     loaded && (
       <>
-        <NavigationContainer>
```

Comment:
Change this back to the NavigationContainer

Reply:
woops i dont think i highlighted all the code properly here. Essentially under this line of code above you have some new code that you were using to test. Revert that back to the normal Navigation Code.

Reply:
+1

Reply:
+1